src/custom/state/claim/hooks/index.ts

```diff
@@ -44,21 +50,39 @@ import {
   setSelectedAll,
   ClaimStatus,
 } from '../actions'
+import { EnhancedUserClaimData } from 'pages/Claim/types'
+import { supportedChainId } from 'utils/supportedChainId'
 
 const CLAIMS_REPO_BRANCH = 'main'
 export const CLAIMS_REPO = `https://raw.githubusercontent.com/gnosis/cow-merkle-drop/${CLAIMS_REPO_BRANCH}/`
 
+// Base amount = 1 VCOW
+const ONE_VCOW = CurrencyAmount.fromRawAmount(
+  V_COW[SupportedChainId.RINKEBY],
+  parseUnits('1', V_COW[SupportedChainId.RINKEBY].decimals).toString()
+)
 // TODO: these values came from the test contract, might be different on real deployment
 // Network variable price
-export const NATIVE_TOKEN_PRICE = {
+export const NATIVE_TOKEN_PRICE: { [chain in SupportedChainId]: string } = {
   [SupportedChainId.MAINNET]: '37500000000000', // '0.0000375' WETH (18 decimals) per vCOW, in wei
   [SupportedChainId.RINKEBY]: '37500000000000', // assuming Rinkeby has same price as Mainnet
   [SupportedChainId.XDAI]: '150000000000000000', // TODO: wild guess, wxDAI is same price as USDC
 }
+
 // Same on all networks. Actually, likely available only on Mainnet (and Rinkeby)
 export const GNO_PRICE = '375000000000000' // '0.000375' GNO (18 decimals) per vCOW, in atoms
 export const USDC_PRICE = '150000' // '0.15' USDC (6 decimals) per vCOW, in atoms
 
+// Symbols of native tokens so we can use this in the UI
+export const NATIVE_TOKEN_SYMBOL: { [chain in SupportedChainId]: string } = {
+  [SupportedChainId.MAINNET]: 'ETH',
+  [SupportedChainId.RINKEBY]: 'ETH',
+  [SupportedChainId.XDAI]: 'XDAI',
+}
```

Comment:
I'm pretty sure that already exists, no?

https://github.com/gnosis/cowswap/blob/2bda3c063326dc79064f132da552f6d430ab4930/src/custom/utils/xdai/hack.ts#L10

Reply:
there it is! was looking for that

Reply:
actually it wasn't even used anymore